thespian/system/transport/TCPTransport.py

```diff
@@ -882,10 +881,8 @@ def _runWithExpiry(self, incomingHandler):
 
             wrecv = list(filter(None, wrecv))
             wsend = list(filter(None, wsend))
-            wrecv.extend(list(filter(None,
-                                     [I
-                                      for I in self._incomingSockets
-                                      if not self._incomingSockets[I].backoffPause()])))
+            wrecv.extend([ I for I in self._incomingSockets if I and
```

Comment:
Removed the redundancy of using the identity filter on a generator that's already essentially doing a filter.  And the result of a generator in `[]`s is a list, so no need to cast it explicitly.